reporting/src/request/steps/check-context.js

```diff
@@ -13,6 +13,9 @@ export function checkValidContext(state) {
   if (!state.page || !state.tabUrlParts || !state.urlParts) {
     return false;
   }
+  if (state.type === 'other') {
```

Comment:
Since it is a subtle problem, would be worth leaving a comment with context. One thing is to mention "prefetch", the other thing would be relevant is where the "type" comes from. I assume, it comes from a browser API. In that case, that would be useful to know which API, and how the field is called there.

I assume, it is the resource type from the webRequest API when the ResourceType equals ["other"](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/ResourceType#other):

>
> other
>
>    Resources that aren't covered by any other available type.

Though from the documentation alone, this is quite unspecific. I wonder what falls into this class.